lib/config.js

```diff
@@ -11,6 +11,7 @@ var fs = require('fs');
 var open = require('fs').openSync;
 var path = require('path');
 var sendMetrics = require('./metrics');
+var sendExpressMetrics = require('./express-metrics');
```

Comment:
I think `express-record` and `sendExpressRecord()` would make more sense for this. We use metrics in various places in docs and code to refer specifically to data that can be expressed as key/value pairs sendable using statsd, and other forms, and to say things like "a metrics license is required". The word "metrics" isn't used for other purposes, and it would be better not to start.